### PR TITLE
fix(twitter): twitter is now X

### LINKED ISSUE #74: Dark theme support
image

### PR DESCRIPTION
**Pull request by skjnldsv**
Fix #74 

- Renamed Twitter to X
- Fixed translation string using wrong app id
- Support dark theme

| Dark | Light |
|:---------:|:------:|
|||

### CODE REVIEW

File: socialsharing_diaspora/css/socialsharing_diaspora.css [hunk 1]
```diff
@@ -1,3 +1,14 @@
-.icon-social-diaspora {
- background-image: url('../img/diaspora.svg');
+.icon-social-diaspora,
+[data-themes*=light] .icon-social-diaspora {
+	background-image: url('../img/diaspora.svg');
+}
+
+[data-themes*=dark] .icon-social-diaspora {
```
Comment (nickvergessen) [inline socialsharing_diaspora/css/socialsharing_diaspora.css:6]: Ah, nice workaround :)
This `[data-themes*=dark]` + `@media (prefers-color-scheme: dark) {` seems to replace the hack we use everywhere :D

**Review by nickvergessen — APPROVED** [review]

File: socialsharing_diaspora/css/socialsharing_diaspora.css (hunk 1 above)
Comment (skjnldsv) [reply socialsharing_diaspora/css/socialsharing_diaspora.css:6]: Don't use it!
That's what the `dist/icons.css` do in server.

Comment (skjnldsv) [reply socialsharing_diaspora/css/socialsharing_diaspora.css:6]: > the hack we use everywhere

Please define "everywhere" ? :see_no_evil: 
This should really not be done :(

Comment (nickvergessen) [reply socialsharing_diaspora/css/socialsharing_diaspora.css:6]: Well not sure which is better? :P

https://github.com/nextcloud/spreed/blob/b526931253c441071d8099d158b916ca2f4744c0/src/utils/isDarkTheme.js#L28-L32

used to load room avatars with the correct background:
https://github.com/nextcloud/spreed/blob/b526931253c441071d8099d158b916ca2f4744c0/src/components/NewMessage/NewMessage.vue#L733

Comment (skjnldsv) [reply socialsharing_diaspora/css/socialsharing_diaspora.css:6]: Alright, not too much used I guess
https://github.com/search?q=repo%3Anextcloud%2Fspreed%20isDarkTheme&type=code

**Pull request merged**